core/web/jsonld/ArticleTrait.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace luya\web\jsonld;
+
+/**
+ * JsonLd - Article Trait
+ *
+ * @see http://schema.org/Article
+ *
+ * @author Alex Schmid <alex.schmid@stud.unibas.ch>
+ * @since 1.0.1
+ */
+trait ArticleTrait
+{
+    use CreativeWorkTrait;
+
+    /**
+     * The actual body of the article.
+     *
+     * @var string
+     */
+    private $_articleBody;
+
+    /**
+     * @return string
+     */
+    public function getArticleBody()
+    {
+        return $this->_articleBody;
+    }
+
+    /**
+     * @param string $articleBody
+     * @return Article|ArticleTrait
+     */
+    public function setArticleBody($articleBody)
+    {
+        $this->_articleBody = $articleBody;
+        return $this;
+    }
+
+    /**
+     * Articles may belong to one or more 'sections' in a magazine or newspaper, such as Sports, Lifestyle, etc.
+     *
+     * @var string
+     */
+    private $_articleSection;
+
+    /**
+     * @return string
+     */
+    public function getArticleSection()
+    {
+        return $this->_articleSection;
+    }
+
+    /**
+     * @param string $articleSection
+     * @return Article|ArticleTrait
```

Comment:
The phpdoc return except an object of type. So we could typehint this for $articleSection like `Article $articleSection` - what do you think? or are there any other optiosn available?

Reply:
I think the hinting right now makes more sense, as return "Article $articleTrait" could confuse in case of what is returned. The return is the article (or any object when the trait is used). Or what do you think?